lighthouse-core/audits/dobetterweb/uses-optimized-images.js

```diff
@@ -116,7 +115,9 @@ class UsesOptimizedImages extends Audit {
 
       totalWastedBytes += webpSavings.bytes;
       results.push({
-        url,
+        url: URL.getDisplayName(image.url),
+        previewUrl: image.url,
+        mimeType: image.mimeType,
```

Comment:
instead of mimeType you _could_ use resourceType. The difference is that it'll report Image even if a jpg's mimetype was plaintext.

```js
record.resourceType() == Common.resourceTypes.Image;
```

but looking closer...
 this would have to be done back in the gatherer. and we're already using mimetype to determine png/jpg anyway... So i guess it's moot in this case. Something to remember for the next one. :)

cc @patrickhulce